### PR TITLE
Reformat and add cross-references

### PR DESCRIPTION
**Pull request by peterfpeterson**
Add links to various items in the `rst` files for `CreateCacheFilename` and `ClearCache`.

**To test:**

Build the docs and see that there are links.

*There is no associated issue.*

*Does not need to be in the release notes* since it is a minor documentation fix.

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####

- [x] Is the code of an acceptable quality?
- [ ] Does the code conform to the [coding standards](http://www.mantidproject.org/Coding_Standards)? Is it well structured with small focussed classes/methods/functions?
- [ ] Are there unit/system tests in place? Are the unit tests small and test the a class in isolation?
- [ ] If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- [x] Do changes function as described? Add comments below that describe the tests performed?
- [ ] How do the changes handle unexpected situations, e.g. bad input?
- [x] Has the relevant documentation been added/updated?
- [x] Is user-facing documentation written in a user-friendly manner?
- [ ] Has developer documentation been updated if required?

- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

File: docs/source/algorithms/CreateCacheFilename-v1.rst [hunk 1]
```diff
@@ -9,50 +9,52 @@
...
-  given property_manager, or it can be glob pattern to match prop
-  names too. but empty list means taking all properties 
-  from the property_manager
-* other_properties: a list of strings. each string is in the form of
-  "key=value" for one property not managed by the property_manager.
+A filename with the form ``<location>/<prefix>_<sha1>.nxs`` will be
+returned as the output property.  If no prefix is specified then file
+result will be ``<location>/<sha1>.nxs``.
+
+* ``property_manager``: an instance of
```
Comment (soininen) [inline docs/source/algorithms/CreateCacheFilename-v1.rst]: Are the items in this list the properties of the algorithm? In that case, the underscores could be changed to CamelCase, i.e. `property_manager` could be `PropertyManager` and so on.

**Review by soininen — CHANGES_REQUESTED** [review]

**Review by soininen — APPROVED** [review]

**Comment by soininen** [comment]
What's with the 'new' sphinx warnings in Ubuntu + Doc Tests? They don't seem to be connected to this PR in any way.

**Comment by peterfpeterson** [comment]
They are in `master` and "somebody" is working on them.

**Comment by soininen** [comment]
Everything seems to be OK :shipit:

**Pull request merged**